zookeeper-api/zookeeper-api-1.0.1-SNAPSHOT.ivy

```diff
@@ -43,9 +43,10 @@ under the License.
     <dependency org="org.slf4j" name="slf4j-log4j12" rev="1.7.14" force="true" conf="compile->compile(*),master(*);runtime->runtime(*)">
         <artifact name="slf4j-log4j12" ext="jar"/>
     </dependency>
-		<dependency org="org.codehaus.jackson" name="jackson-core-asl" rev="1.8.5" force="true" conf="compile->compile(*),master(*);runtime->runtime(*)"/>
-		<dependency org="org.codehaus.jackson" name="jackson-mapper-asl" rev="1.8.5" force="true" conf="compile->compile(*),master(*);runtime->runtime(*)"/>
-		<dependency org="commons-cli" name="commons-cli" rev="1.2" force="true" conf="compile->compile(*),master(*);runtime->runtime(*)"/>
-		<dependency org="org.apache.httpcomponents" name="httpclient" rev="4.5.8" force="true" conf="compile->compile(*),master(*);runtime->runtime(*)"/>
+    <dependency org="org.apache.helix" name="metrics-common" rev="1.0.1-SNAPSHOT" force="true" conf="compile->compile(*),master(*);runtime->runtime(*)"/>
+    <dependency org="org.apache.helix" name="metadata-store-directory-common" rev="1.0.1-SNAPSHOT" force="true" conf="compile->compile(*),master(*);runtime->runtime(*)"/>
+    <dependency org="org.apache.zookeeper" name="zookeeper" rev="3.4.13" force="true" conf="compile->compile(*),master(*);runtime->runtime(*)"/>
+    <dependency org="org.codehaus.jackson" name="jackson-mapper-asl" rev="1.9.13" force="true" conf="compile->compile(*),master(*);runtime->runtime(*)"/>
+    <dependency org="commons-codec" name="commons-codec" rev="1.14" conf="compile->compile(default);runtime->runtime(default);default->default"/>
```

Comment:
Probably not necessary for the same reason for metrics-core above.

Reply:
It is to make sure jars built with ivy have the same dependencies with jars built with maven pom.